### PR TITLE
add replay messages methods

### PR DESCRIPTION
**Pull request by lhaber9**

### CODE REVIEW

File: cloud/main.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ Parse.Cloud.define("hello", function(request, response) {
     response.success(getAliasName());
 });
 
+Parse.Cloud.define("replayMessages", function(request, response) {
+    var requiredParams = ["chatRoomId", "subkey", "count"];
```
Comment (tonyjhuang) [inline cloud/main.js:25]: shouldnt this have some sort of timetoken param too?

See https://www.pubnub.com/docs/web-javascript/storage-and-history#paging

File: cloud/main.js [hunk 2]
```diff
@@ -17,6 +17,17 @@ Parse.Cloud.define("hello", function(request, response) {
     response.success(getAliasName());
 });
```
Comment (tonyjhuang) [inline cloud/main.js:19]: Can you add a comment here showing what the response payload looks like?

File: cloud/main.js (hunk 1 above)
Comment (lhaber9) [inline cloud/main.js:25]: im not sure i understand, do you mean the fact that pubnub can limit replayed messages based on start and end times also?  yea, i didnt add anything for this. You think i should? i cant really think of any time we'd use it...

**Comment by tonyjhuang** [comment]
Hey so just circling back to this, we'll need to return presence indicators as well to replayMessages

**Comment by tonyjhuang** [comment]
Hey what happened with this PR?

**Comment by lhaber9** [comment]
I left it to setup the presence event stuff, ill close and have a new one later today

**Pull request closed without merging**